cranelift/codegen/src/isa/x64/lower.isle

```diff
@@ -573,14 +575,17 @@
 
 ;; 16x8, 32x4, and 64x2 shifts can each use a single instruction.
```

Comment:
This comment is slightly out of date now ("can each use a single instruction, once the shift amount is masked")